src/View/View.php

```diff
@@ -1255,11 +1255,14 @@ protected function _paths($plugin = null, $cached = true)
             }
         }
 
+        $class = new \ReflectionClass('\Cake\View\View');
+        $cake = dirname(dirname($class->getFilename()));
```

Comment:
I think the PR should be just closed. Replacing a constant with reflection isn't very nice very there's nothing wrong with the code.

There isn't much we can do if people overlook PHP basics :)